### DISCUSSION TITLE
Android performance slow. Thermal throttling?

### DISCUSSION THREAD

**Discussion by apple2ms** [tesla-android/issue-tracker#297] [Q&A]
Hi, 
I have tested it and it works, gps and lte modem also work, but the pi seems to be a bit slow. I am using Aluminium Heatsink Case with no fan. Is it possible to check if its throttling? Maybe use an android app?

**Comment by mikegapinski** [comment]
The metal case is blocking the Wi-Fi signal, that is the second problem after the lack of active cooling

**Reply by apple2ms** [reply]
Looking at the cpu temps, they are less than 60 degrees, I don't mind the low fps due to wifi but waze takes a long time to load... Is overclocking possible? 
Also is the fan speed on argon fan automatic? Which cooler should I use to get automatic fan control?
